Included Dependabot

nathannaveen opened the pull request:
**What changes were proposed in this pull request?**

Enable dependabot to get security updates and if needed version updates on dependencies.

**Why are the changes needed?**

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically

Having knowledge about vulnerabilities of the dependencies helps the project owners decide on their dependency's security posture to make decisions.

If the project decides to get updates only on security updates and not on any version updates then setting these options would not open any PR's open-pull-requests-limit: 0

This option has to be enabled in the security section of the project.
https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/configuring-dependabot-security-updates#managing-dependabot-security-updates-for-your-repositories

nathan-415 commented:
@dvyukov A friendly reminder that I have added the license header, could you please approve running the actions?

dvyukov commented:
CI still fails and the PR is unmergable:
https://github.com/google/syzkaller/runs/4089708951?check_suite_focus=true
```
./tools/check-commits.sh
Error: Wrong commit subject format: 'Updated Dependabot'. Please use 'main/affected/package: short change description'. See docs/contributing.md for details.
Error: Wrong commit subject format: 'Added “mod: do” to the commit message prefix.'. Please use 'main/affected/package: short change description'. See docs/contributing.md for details.
./tools/check-copyright.sh
Error: Wrong commit subject format: 'Included Dependabot'. Please use 'main/affected/package: short change description'. See docs/contributing.md for details.
```

nathannaveen commented:
> CI still fails and the PR is unmergable: https://github.com/google/syzkaller/runs/4089708951?check_suite_focus=true
> 
> ```
> ./tools/check-commits.sh
> Error: Wrong commit subject format: 'Updated Dependabot'. Please use 'main/affected/package: short change description'. See docs/contributing.md for details.
> Error: Wrong commit subject format: 'Added “mod: do” to the commit message prefix.'. Please use 'main/affected/package: short change description'. See docs/contributing.md for details.
> ./tools/check-copyright.sh
> Error: Wrong commit subject format: 'Included Dependabot'. Please use 'main/affected/package: short change description'. See docs/contributing.md for details.
> ```

@dvyukov I apologize for the delay. I have fixed the commit message and squashed my commits. Please let me know if there is anything else. Thank You!
